envbuilder_internal_test.go

```diff
@@ -36,10 +33,7 @@ func TestFindDevcontainerJSON(t *testing.T) {
 		require.NoError(t, err)
 
 		// when
-		_, _, err = findDevcontainerJSON(Options{
-			Filesystem:      fs,
-			WorkspaceFolder: "/workspace",
-		})
+		_, _, err = findDevcontainerJSON(&Config{WorkspaceFolder: "/workspace"}, fs, nil)
```

Comment:
I don't mind configuring a logger instance for testing purposes :)

Reply:
Not sure what it means...

Reply:
I meant that you can use a real logger here too:

not sure if exactly this API, but I believe you get the concept :)
```
logger := logrus.New()
logger.SetLevel(log.DebugLevel)
_, _, err = findDevcontainerJSON(&Config{WorkspaceFolder: "/workspace"}, fs, logger)
```